source/specify-fields-return.txt

```diff
@@ -0,0 +1,155 @@
+the ``name`` field value of ``"Emerald Pub"``. It then uses a projection to
+include only the ``name``, ``cuisine``, and ``borough`` fields from the returned documents.
+
+.. io-code-block::
+    
+   .. input:: /includes/project.java
+      :start-after: start-project-include
+      :end-before: end-project-include
+      :language: java
+
+   .. output::
+
+      {'_id': ObjectId('...'), 'borough': 'Manhattan', 'cuisine': 'American', 'name': 'Emerald Pub'}
+      {'_id': ObjectId('...'), 'borough': 'Queens', 'cuisine': 'American', 'name': 'Emerald Pub'}
+
+When you use a projection to specify fields to include in the return
+document, the ``_id`` field is also included by default. All other fields are
+implicitly excluded. To remove the ``_id`` field from the return
+document, you must :ref:`explicitly exclude it <java-rs-project-remove-id>`.
+
+.. _java-rs-project-remove-id:
+
+Exclude the ``_id`` Field
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When specifying fields to include, you can also exclude the ``_id`` field from
+the returned document.
+
+The following example performs the same query as the preceding example, but
+excludes the ``_id`` field from the projection:
+
+.. io-code-block::
+    
+   .. input:: /includes/project.java
+      :start-after: start-project-include-without-id
+      :end-before: end-project-include-without-id
+      :language: java
+
+   .. output::
+
+      {'borough': 'Manhattan', 'cuisine': 'American', 'name': 'Emerald Pub'}
+      {'borough': 'Queens', 'cuisine': 'American', 'name': 'Emerald Pub'}
+
+Specify Fields to Exclude
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following syntax to specify the fields to exclude from the result:
+
+.. code-block:: java
+
+   projection(fields(exclude("<field name>")))
+
+To use a projection, pass your query filter to the ``find()`` method. Then,
+chain the ``projection()`` method to the ``find()`` method.  The following
+example uses the ``find()`` method to find all restaurants with
+the ``name`` field value of ``"Emerald Pub"``. It then uses a projection to
+exclude the ``grades`` and ``address`` fields from the returned documents.
+
+.. io-code-block::
+    
+   .. input:: /includes/project.java
+      :start-after: start-project-exclude
+      :end-before: end-project-exclude
+      :language: java
+
+   .. output:: 
+
+      {'_id': ObjectId('...'), 'borough': 'Manhattan', 'cuisine': 'American', 'name': 'Emerald Pub', 'restaurant_id': '40367329'}
+      {'_id': ObjectId('...'), 'borough': 'Queens', 'cuisine': 'American',
+      'name': 'Emerald Pub', 'restaurant_id': '40668598'}
+
+When you use a projection to specify which fields to exclude,
+any unspecified fields are implicitly included in the return document.
+
+Troubleshooting
+---------------
+
+The following sections describe errors you might see when using projections.
+
+'Cannot Do Exclusion on Field <field> in Inclusion Projection'
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
S: rename this section to something that summarizes the error
eg. Inclusion and Exclusion Error